Assets/Scripts/AnimationAndMovementController.cs

```diff
@@ -48,9 +48,11 @@ public class AnimationAndMovementController : MonoBehaviour
   float previousHeight;
 
   // Interact
-  public bool isInteractPressed = false;
+  public bool isInteractPressed;
+  InputAction interactAction;
   GameObject interactedObject;
-  bool shouldInteract = false;
+
+  public Dictionary<string, string> interactionBindings = new Dictionary<string, string>();
```

Comment:
we should probably make this private and expose it through a getter if possible, so we can discourage other classes from writing to it.